### PR TITLE
jax-toolbox-triage: improve documentation

### PR DESCRIPTION
**Pull request by olupton**
Improve documentation of the triage tool added in https://github.com/NVIDIA/JAX-Toolbox/pull/793.

### CODE REVIEW

**Review by nouiz — COMMENTED** [review]
Looks pretty good!

File: docs/triage-tool.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+```
+
+Where the final result should be read as saying that the test passes with
+[xla@662eb](https://github.com/openxla/xla/commit/662eb45a17c76df93e5a386929653ae4c1f593da)
+and
+[jax@cd04d](https://github.com/jax-ml/jax/commit/cd04d0f32e854aa754e37e4b676725655a94e731),
+but that if JAX is moved forward to include
+[jax@b164d](https://github.com/jax-ml/jax/commit/b164d67d4a9bd094426ff450fe1f1335d3071d03)
+then the test fails.
+This failure is fixed in XXX.
```
Comment (gpupuck) [inline docs/triage-tool.md:223]: What is this?

File: docs/triage-tool.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
+# Triage tool
+
+`jax-toolbox-triage` is a tool to automate the process of attributing regressions to an
+individual commit of JAX or XLA.
+It takes as input a command that returns an error (non-zero) code when run in "recent"
```
[the remark this answers is not in the document]
Comment (olupton) [reply docs/triage-tool.md:5]: I added another sentence here saying the scripts need to exist in the container already.

In the limitation section I noted that again, and also that skipping commits that do not compile is not currently implemented.

File: docs/triage-tool.md (hunk 1 above)
Comment (olupton) [reply docs/triage-tool.md]: I updated it with a link.

**Review by gpupuck — APPROVED** [review]

**Pull request merged**